packager/media/file/udp_file.h

```diff
@@ -14,6 +14,12 @@
 #include "packager/base/compiler_specific.h"
 #include "packager/media/file/file.h"
 
+#if defined(OS_WIN)
+#include <winsock2.h>
+#elif ! defined(SOCKET)
```

Comment:
This condition check should be removed. SOCKET should not have been defined. If it is defined and it is not the same as "int", then we have a serious problem here; it is better to let the compiler capture the problem early in that case.